### DISCUSSION TITLE
Issue with Fetching Data Blob from Performance Logs

### DISCUSSION THREAD

**Discussion by benfrankie** [seleniumbase/SeleniumBase#2925] [General]
Hi Michael, 

I'm encountering an issue with fetching the data blob URL from performance logs while attempting to solve FunCaptcha on a website. Here are the details:

Environment Setup:

Using SeleniumBase with the following driver options

```
Driver(
    headless=True,
    undetectable=True,
    incognito=True,
    log_cdp_events=True
)
```

Steps Taken:

Navigate to the website and log performance logs.
Extract the data blob URL from the logs.
Use the data blob to solve FunCaptcha using an external service.

Problem:

Performance logs are being fetched correctly.
The data blob URL is not being identified from the logs.
Sample log entries indicate various URLs but no blob URL.
The blob URL appears in the network tab but it doesn't get fetched in the performance logs.

Logs:
```
INFO:__main__:Fetching performance logs.
INFO:__main__:Fetched 359 performance logs
INFO:__main__:Checking URL: [various URLs including resources and APIs]
ERROR:__main__:No blob URL found in performance logs.
ERROR:__main__:Failed to fetch the data blob.
```
Code for Fetching Data Blob:

```
def fetch_data_blob(driver):
    logger.info("Fetching performance logs.")
    time.sleep(10)  # Increase sleep duration to ensure logs are captured

    logs = driver.get_log('performance')
    logger.info(f"Fetched {len(logs)} performance logs")

    for entry in logs:
        try:
            log = json.loads(entry['message'])['message']
            if 'Network.responseReceived' in log['method'] and 'params' in log and 'response' in log['params']:
                url = log['params']['response']['url']
                logger.info(f"Checking URL: {url}")
                if 'blob' in url:
                    logger.info(f"Found blob URL: {url}")
                    return url.split('/')[-1]  # Extract the blob value from the URL
        except Exception as e:
            logger.error(f"Error processing log entry: {e}")

    logger.error("No blob URL found in performance logs.")
    return None
```

Despite increasing the sleep duration to ensure logs are captured, the blob URL is not found. I would appreciate any guidance or suggestions on how to reliably capture and identify the data blob URL required for solving FunCaptcha.

Thank you for your help!

**Comment by alex4845** [comment]
I have the same problem. I used wire=True scheme
